packages/core/discovery.config.default.js

```diff
@@ -100,5 +100,6 @@ module.exports = {
     cypressVersion: 12,
     enableCypressExtension: false,
     noRobots: false,
+    preact: false,
```

Comment:
question: double checking - it doesn't affect the current stores that don't have this `preact: false` explicitly, right?

Reply:
I only enable preact if this property is true. `false` and `undefined` should not impact.

Reply:
Great!